lighthouse-core/report/v2/renderer/report-renderer.js

```diff
@@ -127,7 +127,7 @@ class ReportRenderer {
       this._dom.find('.leftnav-item__category', navItem).textContent = category.name;
       const score = this._dom.find('.leftnav-item__score', navItem);
       score.classList.add(`lh-score__value--${Util.calculateRating(category.score)}`);
-      score.textContent = Math.round(Util.formatNumber(category.score));
+      score.textContent = Util.formatNumber(Math.round(category.score));
```

Comment:
Agreed about a consistent calculation of the final category score.  We have a couple of places where you call `Math.round(score)`. 

That said, this nav score should be the same as the [others](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/report/v2/renderer/category-renderer.js#L105). Those just use `Math.round`.

`Math.round(category.score)`